appinventor/components/src/com/google/appinventor/components/runtime/ChartView.java

```diff
@@ -82,6 +92,8 @@ public void Refresh() {
         //   in sequence and then refresh the Chart only once
         // * Using FutureTasks, Hardware Acceleration off (to make invalidate non-async) and run on UI thread
         //   and FutureTask.get() to wait for refreshing to finish
+        // * Using FutureTasks and cancelling current Refresh tasks, and starting another one (+ CountDownLatch
+        //   approach combined with FutureTask cancelling)
```

Comment:
Revise comments and refactor Refresh() method to be applicable for non-async/non-data series refreshing

Reply:
Also remove old references (e.g. RefreshRunnable, getRefreshRunnable method, etc.)